microdnf.spec

```diff
@@ -7,7 +7,7 @@ Summary:        Lightweight implementation of DNF in C
 
 License:        GPLv2+
 URL:            https://github.com/rpm-software-management/microdnf
-Source0:        %{url}/archive/v%{version}/%{name}-%{version}.tar.gz
+Source0:        https://github.com/rpm-software-management/microdnf/archive/%{version}.tar.gz?/%{name}-%{version}.tar.gz
```

Comment:
This is the wrong change to make. You just needed to change it to `%{url}/archive/%{version}/%{name}-%{version}.tar.gz`

Reply:
It is not wrong, you just don't like it ;) For example I don't like %{url} because it takes more time to copy-paste the URL, especially when it is not GitHub, but a subdirectory with tarballs. Restored %url and force-pushed.